.github/workflows/pull_request.yml

```diff
@@ -6,6 +6,7 @@ on:
       - master
   pull_request:
     types: [opened, synchronize, reopened]
+  workflow_dispatch:
```

Comment:
Why do you need to add a workflow_dispatch condition?

Reply:
That's not needed. removed.